lessons/pydata/ml_intro/index.ipynb

```diff
@@ -0,0 +1,492 @@
+{
```

Comment:
<p>Proč nepoužijeme <code>train_test_split</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
taky opravím, ať je to stejné jako u klasifikace